src/commands/config/index.js

```diff
@@ -16,6 +16,9 @@ const config = () => {
     configurationVault.setScopePrompt(
       answers[CONFIGURATION_PROMPT_NAMES.SCOPE_PROMPT]
     )
+    configurationVault.setGitmojisUrl(
+      answers[CONFIGURATION_PROMPT_NAMES.GITMOJIS_URL]
+    )
```

Comment:
Great catch! Can you update the test spec to include a case for covering this?

https://github.com/carloscuesta/gitmoji-cli/blob/master/test/commands/config.spec.js

Reply:
Thanks for the comment! :+1: